core/src/main/res/values/strings.xml

```diff
@@ -624,118 +624,118 @@
     <string name="feed_tags_label">Tags</string>
-    <string name="feed_tags_summary">Change the tags of this podcast to help organize your subscriptions</string>
+    <string name="feed_tags_summary">Tag your subscriptions to organize them.
     <string name="feed_folders_include_root">Show in main list</string>
-    <string name="auto_download_settings_label">Auto Download Settings</string>
+    <string name="auto_download_settings_label">Auto-downloading</string>
     <string name="episode_filters_label">Episode Filter</string>
-    <string name="episode_filters_description">List of terms used to decide if an episode should be included or excluded when auto downloading</string>
+    <string name="episode_filters_description">Terms to prevent or auto-download episodes</string>
     <string name="episode_filters_include">Include</string>
     <string name="episode_filters_exclude">Exclude</string>
     <string name="episode_filters_hint">Single words \n\"Multiple Words\"</string>
     <string name="keep_updated">Keep Updated</string>
     <string name="keep_updated_summary">Include this podcast when (auto-)refreshing all podcasts</string>
-    <string name="auto_download_disabled_globally">Auto download is disabled in the main AntennaPod settings</string>
+    <string name="auto_download_disabled_globally">Auto-downloading turned of in the main settings</string>
     <string name="statistics_listened_for">Listened for:</string>
     <string name="statistics_episodes_on_device">Episodes on the device:</string>
     <string name="statistics_space_used">Space used:</string>
-    <string name="statistics_view_all">View for all podcasts »</string>
+    <string name="statistics_view_all">For all podcasts »</string>
 
     <!-- AntennaPodSP -->
-    <string name="sp_apps_importing_feeds_msg">Importing subscriptions from single-purpose apps&#8230;</string>
+    <string name="sp_apps_importing_feeds_msg">Importing subscriptions from single-purpose apps…</string>
 
     <!-- Add podcast fragment -->
-    <string name="search_podcast_hint">Search podcast…</string>
-    <string name="search_itunes_label">Search iTunes</string>
-    <string name="search_podcastindex_label">Search Podcastindex.org</string>
-    <string name="search_fyyd_label">Search fyyd</string>
+    <string name="search_podcast_hint">Search for podcast…</string>
+    <string name="search_itunes_label">Search on iTunes</string>
+    <string name="search_podcastindex_label">Search on Podcastindex.org</string>
+    <string name="search_fyyd_label">Search on fyyd</string>
     <string name="advanced">Advanced</string>
-    <string name="add_podcast_by_url">Add Podcast by RSS address</string>
-    <string name="add_podcast_by_url_hint" translatable="false">www.example.com/feed</string>
+    <string name="add_podcast_by_url">Add podcast RSS address</string>
+    <string name="add_podcast_by_url_hint">www.example.com/feed</string>
```

Comment:
It is deliberately not translatable

Reply:
/feed/ should be. Some places have other URLs reserved for examples.

Reply:
Should be possible to make this www.localname.toplevel/feed-in-local-language